lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -3024,6 +3029,30 @@ calculateFeePercentiles
             ) -> pure $ Fee requiredCost
         e -> throwE e
 
+-- | Make a pair of fee estimation percentiles more imprecise. Useful for hiding
+-- small differences between the estimation and actual tx construction when
+-- integration tests expect them to coincide.
```

Comment:
> Useful for hiding small differences between the estimation and actual tx construction when integration tests expect them to coincide.

Perhaps the integration tests should **_not_** expect them to coincide? It seems to me that this kind of expectation is fundamentally broken. 😄 

If we do want to add "padding", then perhaps the correct place to add that padding would be in the integration test suite itself? (Or otherwise adjust the integration tests so that they **_don't_** check this kind of thing: to my mind, perhaps the best place for testing the estimation logic would be with property tests.)